Content.Shared/_DV/CosmicCult/MonumentOnDespawnComponent.cs

```diff
@@ -0,0 +1,17 @@
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Shared._DV.CosmicCult;
```

Comment:
Remove unused using directive

```suggestion
using Robust.Shared.Prototypes;

namespace Content.Shared._DV.CosmicCult;
```